hack/local-up-cluster.sh

```diff
@@ -812,17 +813,18 @@ EOF
       # Convert from foo=true,bar=false to
       #   foo: true
       #   bar: false
-      for gate in $(echo ${FEATURE_GATES} | tr ',' ' '); do
-        echo ${gate} | ${SED} -e 's/\(.*\)=\(.*\)/  \1: \2/'
+      for gate in $(echo "${FEATURE_GATES}" | tr ',' ' '); do
+        echo "${gate}" | ${SED} -e 's/\(.*\)=\(.*\)/  \1: \2/'
       done
     fi >>/tmp/kube-proxy.yaml
 
     if [[ "${REUSE_CERTS}" != true ]]; then
         generate_kubeproxy_certs
     fi
 
+    # shellcheck disable=SC2024
```

Comment:
I think it's a redirect to the normal user.
So I ignore this message.

In ./hack/local-up-cluster.sh line 827:
      --master="https://${API_HOST}:${API_SECURE_PORT}" >"${PROXY_LOG}" 2>&1 &
                                                        ^-- SC2024: sudo doesn't affect redirects. Use ..| sudo tee file